src/components/app/content/cards/card/index.js

```diff
@@ -1,16 +1,21 @@
-import React, {Fragment, useState} from 'react';
+import React, {Fragment, useEffect, useState} from 'react';
 import './index.css'
 
 const classNames = require('classnames');
 
-function Card() {
+function Card({readonly}) {
     const initHeader = 'Caption';
     const initBody = 'Text...';
```

Comment:
Your component should be out of any pre-defined data, otherwise you could not fully handle its re-usage. If you want any data (even initial) you should better store it in the app state and pass as prop.